composer.json

```diff
@@ -33,43 +33,43 @@
         "knplabs/knp-menu": "^3.1",
         "knplabs/knp-menu-bundle": "^3.0",
         "psr/container": "^1.0",
-        "psr/log": "^1.1",
+        "psr/log": "^1.1 || ^2.0",
```

Comment:
required first a release of cache library

Reply:
Can this be done on 3.x too ?

Reply:
Yes: https://github.com/sonata-project/SonataAdminBundle/pull/7356